eng/common/testproxy/test-proxy-docker.yml

```diff
@@ -6,6 +6,18 @@ steps:
         $(Build.SourcesDirectory)/eng/common/scripts/trust-proxy-certificate.ps1
     displayName: 'Language Specific Certificate Trust'
 
+  # windows has a startup bug where the service is unavailable. See Azure/azure-sdk-tools#2122
+  - pwsh: |
+        try {
+          docker info
```

Comment:
This won't work, external process call exit codes do not trigger try/catch. You need to handle the non-zero $LASTEXITCODE manually.

Reply:
Unless somehow all this stuff behaves differently on Windows, which I'm starting to suspect based on your lower changes.

Reply:
Trying to get the issue reproing like:

![image](https://user-images.githubusercontent.com/45376673/140405649-1a3c0317-e598-4d20-8d89-656bd4823d3b.png)

No failures. Because of course not.